gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -168,6 +177,15 @@
             createCameraUpdateFlowable(locationLockStateFlowable));
     this.projectLoadingState =
         LiveDataReactiveStreams.fromPublisher(projectRepository.getProjectLoadingState());
+    this.layers =
+        LiveDataReactiveStreams.fromPublisher(
+            projectRepository
+                .getProjectLoadingState()
+                .map(
+                    loadableProject ->
+                        projectRepository.getModifiableLayers(
```

Comment:
I think we should show all layers in the dialog, modifiable or otherwise. You can get these directly from the Proejct object.

Reply:
Sure.